Item meta exception after spigot 1.8 support

```
ERROR  [13:12:54 ERROR]: Could not pass event ProjectileLaunchEvent to PGM v0.12-SNAPSHOT (git-a3992e8)
org.bukkit.event.EventException
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) ~[sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:74) ~[sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:65) ~[sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at org.bukkit.plugin.SimplePluginManager.callEventHandler(SimplePluginManager.java:569) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at org.bukkit.plugin.SimplePluginManager.dispatchEvent(SimplePluginManager.java:534) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at org.bukkit.plugin.SimplePluginManager.callEventSynchronously(SimplePluginManager.java:527) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:521) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:496) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at org.bukkit.craftbukkit.v1_8_R3.event.CraftEventFactory.callProjectileLaunchEvent(CraftEventFactory.java:793) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.World.addEntity(World.java:1073) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.World.addEntity(World.java:1028) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.ItemBow.a(ItemBow.java:68) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.ItemStack.b(ItemStack.java:533) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.EntityHuman.bU(EntityHuman.java:113) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:631) [?:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.PacketPlayInBlockDig.a(SourceFile:40) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.PacketPlayInBlockDig.a(SourceFile:10) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_282]
	at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.MinecraftServer.processTasks(MinecraftServer.java:1726) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.DedicatedServer.processTasks(DedicatedServer.java:397) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:884) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:823) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:724) [sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_282]
Caused by: java.lang.NullPointerException
	at tc.oc.pgm.util.inventory.tag.LegacyItemTag.get(LegacyItemTag.java:46) ~[?:?]
	at tc.oc.pgm.util.inventory.tag.LegacyItemTag.get(LegacyItemTag.java:26) ~[?:?]
	at tc.oc.pgm.kits.tag.Grenade$Tag.get(Grenade.java:30) ~[?:?]
	at tc.oc.pgm.kits.tag.Grenade$Tag.get(Grenade.java:23) ~[?:?]
	at tc.oc.pgm.kits.KitMatchModule.onGrenadeLaunch(KitMatchModule.java:134) ~[?:?]
	at sun.reflect.GeneratedMethodAccessor185.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_282]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_282]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:300) ~[sportpaper.jar:git-SportPaper-v1.0-19-g2ad6179]
	... 26 more
```
Unsure under which condition this occurs, but the lore manipulation that LegacyItemTag does seems to have changed behavior in the 1.8 spigot support patch, and grenades throw an exception.

I assume item.getItemMeta returns a null, and you can't run getLore on it

--- a/core/src/main/java/tc/oc/pgm/kits/KitMatchModule.java
+++ b/core/src/main/java/tc/oc/pgm/kits/KitMatchModule.java
@@ -145,6 +145,7 @@ public class KitMatchModule implements MatchModule, Listener {
       Grenade grenade = Grenade.get(event.getEntity());
       if (grenade != null) {
         event
+            .getEntity()
             .getWorld()
             .createExplosion(
                 event.getEntity(),

--- a/util/src/main/java/tc/oc/pgm/util/inventory/tag/LegacyItemTag.java
+++ b/util/src/main/java/tc/oc/pgm/util/inventory/tag/LegacyItemTag.java
@@ -43,6 +43,7 @@ final class LegacyItemTag implements ItemTag<String> {
   @Nullable
   @Override
   public String get(ItemStack item) {
+    if (!item.hasItemMeta()) return null;
     final List<String> lore = item.getItemMeta().getLore();
     if (lore == null || lore.isEmpty()) return null;
 
@@ -52,6 +53,7 @@ final class LegacyItemTag implements ItemTag<String> {
 
   @Override
   public void set(ItemStack item, String value) {
+    if (!item.hasItemMeta()) return;
     ItemMeta itemMeta = item.getItemMeta();
     List<String> lore = itemMeta.getLore();
 
@@ -68,6 +70,7 @@ final class LegacyItemTag implements ItemTag<String> {
 
   @Override
   public void clear(ItemStack item) {
+    if (!item.hasItemMeta()) return;
     ItemMeta itemMeta = item.getItemMeta();
     final List<String> lore = itemMeta.getLore();
     if (lore == null || lore.isEmpty()) return;

